module4/projects/index.md

```diff
@@ -13,3 +13,6 @@ The following projects will be assigned and their completion is required to grad
 ## Professional Development Projects
 - Weeks 1-4 (Solo): [Personal Portfolio](./personal_portfolio/)
 - Weeks 4-5 (Solo): [Employer Case Study](./employer_case_study/)
+
+## Technical Challenges
```

Comment:
I know this lives on the projects page on the current mod 4 site but it also feels a little out of place because it's not a project. That said, I keep ending up back here as the "least bad". This isn't really a question or an action item. I'm just commenting in case anyone else has thought the same thing but actually has an idea of where else it should go.